src/main/java/core/model/match/MatchStatistics.java

```diff
@@ -91,6 +105,7 @@ public int getTrainMinutesPlayedInPositions(int spielerId, int[] accepted) {
 		// Done with substitutions, add end if necessary
 
 		if (inPosition) {
+			// TODO: if opponent is walkover +=90
```

Comment:
is it a todo you intended to implement before submiting the PR ? do you want to open a dedicated ticket for this?

Reply:
this line is left over from my first plan. finally, the treatment of walkover matches is done a few lines above. 
i will remove this line.

Reply:
ok